maidr/widget/shiny.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import annotations
+
+from shiny.render import ui
+from shiny.types import Jsonifiable
+
+import maidr
+
+
+class render_maidr(ui):
```

Comment:
We use `PascalCase` for class name in our package. why do you use `snake_case` here? Is this the Shiny convention?

Reply:
I checked Shiny and there are no such conventions. I have renamed the class name to adhere to `PascalCase`.